docs/api/BottomNavigation/BottomNavigationButton.md

```diff
@@ -0,0 +1,14 @@
+BottomNavigationButton
+=======
+
+
+
+Props
+-----
+
+
+| Name | Type | Default | Description |
+|:-----|:-----|:-----|:-----|
+| className | string |  | The CSS class name of the root element. |
+| label | string |  | Set the label describing the view for this item. |
+| icon | node |  | Set the icon representing the view for this item. |
```

Comment:
Is there a way to set private proptypes to not show in documentation like index for BottomNavigationButton?